architecture/core.rst

```diff
@@ -305,7 +382,7 @@ decision. This means trusted threads are scheduled by the Linux kernel.
 
 TEE core handles a static number of trusted threads, see ``CFG_NUM_THREADS``.
 
-Trusted threads are only expensive on memory constrained system, mainly
+Trusted threads are expensive on memory constrained system, mainly
 regarding the execution stack size.
```

Comment:
`regarding`, what about change to `because of`?

Reply:
OK